main.go

```diff
@@ -339,12 +337,12 @@ func main() {
 		}
 	}
 
-	lkJwtPort := os.Getenv("LIVEKIT_JWT_PORT")
-	if lkJwtPort == "" {
-		lkJwtPort = "8080"
+	lkJwtBind := os.Getenv("LIVEKIT_JWT_BIND")
```

Comment:
It would be nice if we could keep LIVEKIT_JWT_PORT so that this isn't a breaking change, potentially just set `lkJwtBind` to the value of `:` plus port. Otherwise, it's a migration step for existing users